README.md

```diff
@@ -1,4 +0,0 @@
-# mondemand-backend-annotations-elasticsearch
-A plugin for the mondemand server to send annotations to elasic search.
-
-So it is
```

Comment:
It might be useful to include instructions here on any special setup you need for elastic or how to configure this backend.  I've been trying to start doing that more (not super successfully but still trying ;) ).